solutions/instances/main.tf

```diff
@@ -202,3 +202,47 @@ module "scc_wp" {
   access_tags                   = var.scc_workload_protection_access_tags
   scc_wp_service_plan           = var.scc_workload_protection_service_plan
 }
+
+#######################################################################################################################
+# SCC Event Notifications Configuration
+#######################################################################################################################
+
+locals {
+  parsed_existing_en_instance_crn = var.existing_en_crn != null ? split(":", var.existing_en_crn) : []
+  existing_en_guid                = length(local.parsed_existing_kms_instance_crn) > 0 ? local.parsed_existing_kms_instance_crn[7] : null
+}
+
+data "ibm_en_destinations" "en_destinations" {
+  count         = var.existing_en_crn != null ? 1 : 0
+  instance_guid = var.existing_en_guid
```

Comment:
there is no `existing_en_guid` variable. It should be a local value parsed from the CRN input